frontend/device/android/device.lua

```diff
@@ -8,7 +8,7 @@ local function yes() return true end
 local function no() return false end
 
 local Device = Generic:new{
-    model = "Android",
+    model = "Android_" .. android.getProduct(),
```

Comment:
I've been meaning to add something like that. :-)

Reply:
I will leave model = android.getProduct()

Hopefully some day we will need to differenciate between models that have full e-ink support.